src/server/scripts/Kalimdor/RuinsOfAhnQiraj/boss_kurinnaxx.cpp

```diff
@@ -87,6 +84,11 @@ class boss_kurinnaxx : public CreatureScript
 
             events.Update(diff);
 
+            if (me->GetDistance(me->GetVictim()) > 60.00f && me->GetDistance(me->GetVictim()) < 120.00f)
+            {
+                DoCastVictim(SPELL_SUMMON_PLAYER);
+            }
+
```

Comment:
This needs to be double-checked. This spell is usually used by boss to summon unreachable players.